### ISSUE TITLE
RDC load lib err

### ISSUE THREAD

**Issue by joesantoro** [spyder-ide/spyder#15754]
<!---   *** BEFORE SUBMITTING: PASTE CLIPBOARD HERE TO COMPLETE YOUR REPORT ***   ---!>
Run Spyder 5.0 over Win10 remote desktop error126.

**Comment by steff456** [comment]
Hi @joesantoro,

Can you please provide me more information to know what error appeared? Please include the version, screenshots and a series of steps for us to try and reproduce this issue.

Thanks for reporting!

**Comment by joesantoro** [comment]
Hi Stephannie,
I ended up solving the issue using some suggestions I found on
github/stackoverflow and other sites. Here is how I did it.

I was getting this error whilst loading Spyder 5.0 on a remote desktop:

[image: image.png]

I ended up running gpedit (from the command line) on the host computer.

   1. navigate to Computer Configuration->Administrative Templates->Windows
   Components->Remote Desktop Services->Remote Desktop Session Host->Remote
   Session Environment
   2. Enable use hardware graphics adapters for all Remote Desktop Services
   sessions, Prioritize H.264/AVC 444 graphics mode for Remote Desktop
   Connections, amd Configure H.264/AVC hardware encoding for Remote Desktop
   Connections

[image: image.png]

Next open up control panel->Device manager->display adapters and disable
Intel(R) HD/UHD Graphics

[image: image.png]

I think it was this last step that was the most important and that actually
fixed the problem.
Restarting the host computer after making these changes might also help.

I hope this helps.
Joe S.
